routes/index.js

```diff
@@ -288,7 +288,9 @@ function postBadgeIn(req, res, next) {
     else {
       req.messageType = "success";
       req.message = "You successfully Badge In";
+      dbAPI.logEvent("EPL Badge IN", BadgeNumber).then(function (result) {
```

Comment:
I dont think you need the `.then` in this case. You can just remove it and just call then function.